src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+class Deadline extends Task {//Need to be done by deadline
```

Comment:
Should the comments be made as a Header comment for the class (Javadocs)

Reply:
Yeap thanks for pointing it out!